Sources/NFCPassportReader/PassportReader.swift

```diff
@@ -12,11 +12,17 @@ import Foundation
 import UIKit
 import CoreNFC
 
+@available(iOS 15, *)
+public protocol PassportReaderDelegate: AnyObject {
```

Comment:
What would you think about:

```suggestion
public protocol PassportReaderTrackingDelegate: AnyObject {
```

I feel like that more clearly suggests what it is about. From the name `PassportReaderDelegate` one could assume it is a general delegate that also communicates errors and results.

Reply:
Agree! Naming it `PassportReaderTrackingDelegate` makes more sense